src/BenchmarkDotNet/Running/BenchmarkCase.cs

```diff
@@ -26,6 +28,13 @@ private BenchmarkCase(Descriptor descriptor, Job job, ParameterInstances paramet
             Config = config;
         }
 
+        public Runtime GetRuntime()
+        {
+            return Job.Environment.HasValue(EnvironmentMode.RuntimeCharacteristic)
+                              ? Job.Environment.Runtime
+                              : RuntimeInformation.GetCurrentRuntime();
+        }
+
```

Comment:
nit: since other methods in this type use expression bodies, this one could be as well (for consistency)
```suggestion
        public Runtime GetRuntime() => Job.Environment.HasValue(EnvironmentMode.RuntimeCharacteristic)
                ? Job.Environment.Runtime
                : RuntimeInformation.GetCurrentRuntime();

```